comscore/src/integration/ComscoreStreamingAnalyticsTHEOIntegration.ts

```diff
@@ -168,6 +177,16 @@ export class ComscoreStreamingAnalyticsTHEOIntegration {
                     console.log(`[COMSCORE - STATE] State change ${this.state} -> VIDEO`);
                 this.state = ComscoreState.VIDEO;
                 this.setContentMetadata();
+                if (this.dvrWindowLengthMs) {
+                    this.streamingAnalytics.setDvrWindowLength(this.dvrWindowLengthMs);
+                    if (this.configuration.debug && LOG_STREAMINGANALYTICS)
+                        console.log(`[COMSCORE - StreamingAnalytics] setDvrWindowLength ${this.dvrWindowLengthMs}`);
+                    this.streamingAnalytics.startFromDvrWindowOffset(this.dvrWindowOffsetMs ?? 0);
+                    if (this.configuration.debug && LOG_STREAMINGANALYTICS)
+                        console.log(
+                            `[COMSCORE - StreamingAnalytics] startFromDvrWindowOffset ${this.dvrWindowOffsetMs}`
+                        );
+                }
```

Comment:
Can maybe include this in a more general `setContentMetadata` and/or make a separate function for it since it is duplicated below as well?

Reply:
Decided to leave it out of `setContentMetadata` (since it only prepares the metadata object and interacts with only one StreamingAnalyics api) but extracted it to a separate function